data/coalition/heliarch intro.txt

```diff
@@ -0,0 +1,1843 @@
+				accept
+	npc "scan outfits"
+		personality staying uninterested target
+		government "Quarg"
+		system "Enif"
+		fleet
+			names "quarg"
+			variant
+				"Quarg Skylark"
+				"Quarg Wardragon"
+	on complete
+		"coalition jobs" ++
+		payment 200000
+		conversation
+			`Once again the engineers remove your outfit scanner, taking a copy of the scan logs and handing it to the Heliarch agents, who give you <payment>. "Grateful to you, we are, Captain <last>," the Saryd says. "Remember your assistance, the consul will."`
+			`	When the engineers return, they put your outfit scanner back in place. The Heliarch agents say their goodbyes, reassuring you that the Quarg shouldn't attack you for this. "If interested in doing some investigation of your own, you wish, maybe land on their rings again, you could," the Kimek suggests. "Maybe find out something of interest for us, you will."`
+			`	He wishes you safe travels, and leaves with his colleagues.`
+
+mission "Quarg Interrogation"
+	invisible
+	landing
+	deadline 1
+	source
+		government "Quarg"
+		not planet "Humanika"
+	destination "Earth"
+	to offer
+		has "Heliarch Recon 2-B: done"
+	on offer
+		conversation
+			`As you land on <origin>, a Quarg is patiently waiting right outside. Seeing that it won't go away, you head outside to speak with it.`
+			`	"Hello, <first> <last>," it says.`
+			choice
+				`	"Hi, can I help you?"`
+				`	"Is there some sort of problem?"`
+			`	It blinks, then slowly looks to a Wardragon parked farther away. "You were seen landing on some of our worlds and rings, bringing about strange scanning mechanisms, and were also found to be scanning some of our ships. Why is that?"`
+			choice
+				`	"I was just curious about your species."`
+					goto lie
+				`	"What are you talking about? I never did any of that."`
+					goto lie
+				`	"Scanning? I'm sorry, I think my ship just had some malfunctions. I didn't mean to offend."`
+					goto lie
+				`	"The Heliarchs asked me to do it."`
+					goto truth
+			label lie
+			`	Around ten seconds pass without it saying anything, simply staring at your eyes and blinking slowly. "I see." Without another word, it walks away, headed to the Wardragon.`
+				accept
+			label truth
+			`	"And why did you agree? Do you wish to join their ranks, perhaps?" it asks.`
+			choice
+				`	"I just wanted to help. It's not like I destroyed your ships or something."`
+				`	"They told me that the Quarg keep others in 'separate boxes', and that you hinder their development."`
+					goto hinder
+			`	"Perhaps not, but you agreed to collect intel on us and our worlds, which hold countless civilians, and deliver the findings to a group that hates us."`
+				goto end
+			label hinder
+			`	"And you believe that? Have we restricted your movement? Chastised you for interacting with the Hai or Korath? Even after you agreed to help those foolish, tyrannical Heliarchs, are you not here, in one of our worlds, peacefully conversing with a Quarg?"`
```

Comment:
What if you haven't met all of these races?

Reply:
Looking at the code You need to have visited the Korath and Hai ringworlds before Heliarch Recon 1 can be triggered which would reasonably give you some idea of the species. So the only species that applies to is the Hai and that can easily be fixed by adding a "First Contact: Hai" check to Heliarch Recon 1. Or adding an event were you talk to the Quarg on Alta Hai about the Hai when first landing there and using that as a check.